src/main/java/org/openrewrite/github/ReplaceRunners.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.github;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.internal.lang.NonNull;
+import org.openrewrite.yaml.ChangeValue;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = false)
+@Getter
+public class ReplaceRunners extends Recipe {
+    @Option(displayName = "Job Name",
+            description = "The name of the job to update",
+            example = "build")
+    private final String jobName;
+
+    @Option(displayName = "Runners",
+            description = "The new list of runners to set",
+            example = "runner1,runner2"
```

Comment:
Perhaps this example value should not be comma separated, [given this example](https://github.com/openrewrite/rewrite/blob/v7.35.0/rewrite-java/src/main/resources/META-INF/rewrite/cleanup.yml#L58). What do you think?
We can use a singular value to avoid ambiguity.
```suggestion
            example = "ubuntu-latest"
```

Reply:
Thank you. I was unsure how to represent it given that the input is a list